--- debian/rules.old    2024-01-04 09:47:42.632001178 +0000
+++ debian/rules        2024-01-04 09:49:15.547373512 +0000
@@ -79,7 +79,7 @@
        #       (cd debian/$$flavour && ./buildconf && cp ../../ltmain.sh .) \
        #done
        dh_auto_configure -- ${DEB_CONFIGURE_EXTRA_FLAGS} \
-               --with-gnutls
+               --without-gnutls
        cd debian/build-openssl && dh_auto_configure -- ${DEB_CONFIGURE_EXTRA_FLAGS} \
                --with-openssl \
                --enable-ssl-crtd
